main.py

```diff
@@ -49,13 +49,13 @@
     if ARGV.start:
         print('''
         \U0001F604 Hello and Welcome to Dynamic CLI 
-        \U0001F917 Use the following commands to get started 
-        \U0001F50E Search on StackOverflow with '-s' 
-        \U0001F4C4 Open browser to create new Stack Overflow question with '-n [title(optional)]' 
-        \U0001F4C2 Save answer to a file with '-file'
-        \U00002728 Know the version of Dynamic CLI with '-V' 
-        \U0001F609 See this message again with '-st'
-        \U00002755 Get help with '-h'
+        Use the following commands to get started 
+        \U0001F50E to search StackOverflow, type  'dynamic -s' 
```

Comment:
Fix the Indentation of words for the user to read!

Example: 

`\U0001F50E to search StackOverflow, type 'dynamic -s'`

can be replaced by 

`\U0001F50E To Search on StackOverflow, type 'dynamic -s'`


And, similar till #L58